### ISSUE TITLE
Cfgs produced for new Picade screen resolution of 1024x768 display incorrectly on Picade

### ISSUE THREAD

**Issue by brooksoid** [dankcushions/crt-pi-configs#17]
Hi there,

I used crt-pi-configs to create configs for RetroPie running on a Pi, with the new Picade screen size (1024x768), using `python crt_pi_configs.py mame2003 1024 768`

Deploying any of these configs results in a full screen display with the CRT effect in place, but scaled incorrectly so I can only see the top left hand portion of the screen (maybe a third or a quarter...)

Seems likely that I'm doing something wrong, but as the only params are resolution, I can't imagine what...

**Comment by dankcushions** [comment]
what happens if you don't use the cfgs?

**Comment by brooksoid** [comment]
Turns out I was using 1024x768 shaders with either 720p or 1080p resolution. I had made the assumption that Retropie would detect the 1024x768 resolution of my screen, but I think it's been running at 720p or maybe 1080p.

I found a post of yours elsewhere on forcing video modes, and stuck it in my config.txt to force 1024x768. Regenerated the corresponding config files. Everything is fine.

I haven't come across anything in the Picade documentation mentioning setting the resolution when setting up Retropie... going to check appropriate forums and drop a note in case anyone else is running at a higher resolution without realising it.

Thanks for the reply, and the shaders!
